core/src/main/scala/filodb.core/memstore/PartKeyLuceneIndex.scala

```diff
@@ -276,6 +276,10 @@ class PartKeyLuceneIndex(dataset: Dataset,
                            partKeyNumBytes: Int,
                            partId: Int, startTime: Long, endTime: Long): Document = {
     val document = new Document()
+    // TODO We can use RecordSchema.mapify to get the name/value pairs from partKey.
```

Comment:
No we really don't want to do this.  It creates a huge amount of objects.  mapify should NOT be used in any ingestion paths at all, for memory reasons.  Please instead note that we cannot use mapify due to too much memory overhead.